### PR TITLE
connect followers pages with profile page

### PR DESCRIPTION
**Pull request by Tretmax**
image
create links on followers

### CODE REVIEW

**Review by Mykyta-Pysarev — APPROVED** [review]
Looks good.

**Comment by ArtemLigerko** [comment]
Nice! And can you add cursor pointer to 'following' and 'followers' links

**Comment by Tretmax** [comment]
> 

done

File: src/pages/Profile.tsx [hunk 1]
```diff
@@ -68,6 +81,27 @@ const ProfilePage = ({ tabKey }: TabKeyProps) => {
             </StyledButton>
           </div>
         </div>
+        <div className="d-flex pb-4">
+          <UnderlineHover
```
Comment (alexandrtovmach) [inline src/pages/Profile.tsx]: why not to use link?

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Review by ArtemLigerko — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**